rosdep/base.yaml

```diff
@@ -4617,6 +4617,11 @@ nkf:
   fedora: [nkf]
   gentoo: [app-i18n/nkf]
   ubuntu: [nkf]
+nlohmann-json:
+  debian: [nlohmann-json-dev]
+  fedra: [json]
+  gentoo: [nlohmann_json]
+  ubuntu: [nlohmann-json-dev ]
```

Comment:
This looks to only be available in Bionic and later.  Since we still support Xenial, you'll need to break this rule out into a "null" for xenial, and the package name here for the wildcard.  See elsewhere in this file on how to do that.  Also, there is a trailing whitespace in the brackets (that I'm surprised our CI didn't catch).

Reply:
@clalancette historically we've let this slide. It's known not to break anyone who's using the older platform because they are guaranteed not to be using the rule currently and saves us complexity in our management. That was particularly important before we had the wildcard support. Now we do have the ability to do this without incurring the cost of repeatedly adding new rules so I guess it's worth doing now.

Reply:
I fixed this issue in https://github.com/ros/rosdistro/pull/24432/commits/000e1763d563544e2eec3f54a6bba064e2aac610